src/app.ts

```diff
@@ -1,12 +1,26 @@
-import express, { Response } from 'express';
+import express from 'express';
 import dotenv from 'dotenv';
+import router from './routes/v1';
+import globalErrorHandler from './middlewares/errorHandler.middleware';
+import ApiException from './utils/exceptions/ApiException';
+import * as statusCodes from './utils/constants/statusCodes';
 
 // Config
 dotenv.config();
 const app = express();
 
-app.get('/', (req, res: Response) => {
-    return res.status(200).json({ msg: 'Hello World!' });
-});
+// Middleware
+app.use(express.json());
+
+// V1 Routes
+app.use('/api/v1', router);
+
+// Serve frontend
+app.use(express.static('public/client'));
+app.get('*', (req, res) => res.sendFile('public/client/index.html', { root: __dirname }));
+
+// Error Handler
+app.all('*', (req, res, next) => next(new ApiException(statusCodes.NOT_FOUND, "Oops! Looks like you're lost.")));
```

Comment:
@Dhamareshwarakumar I like the way you moved status codes to the constants file how about moving `Oops! Looks like you're lost.` to a constants file like (ResponseMessage)

Reply:
- I am planning to do that but don't have better idea where to place and how to group files.
- I will do it

Reply:
As of know I want to leave as it is. is that okay @sunkarivasu

Reply:
okay